src/interactive/azext_interactive/azclishell/az_lexer.py

```diff
@@ -55,3 +55,13 @@ class ToolbarLexer(RegexLexer):
             (r'.', Operator),
         ]
     }
+
+
+class ScenarioLexer(RegexLexer):
```

Comment:
I am not very clear about the function of this class. Could you please add some comments?